src/xdocs/config_design.xml

```diff
@@ -1571,17 +1751,34 @@ class SomeClass
           Example of allowed fields:
         </p>
         <source>
-class SomeClass
-{
-  @org.junit.Rule
-  public TemporaryFolder publicJUnitRule = new TemporaryFolder(); // no warning
-  @org.junit.ClassRule
-  public static TemporaryFolder publicJUnitClassRule = new TemporaryFolder(); // no warning
-  @com.google.common.annotations.VisibleForTesting
-  public String testString = ""; // no warning
+public final class MyClass {
+ private int myPrivateField1;
+
+ int field1;               // violation, must have a visibility modifier
+
+ protected String field2; // violation, protected visibility is not allowed
+
+ public int field3 = 42; // violation, not static final, immutable, matching the pattern
+ public long serial = 1L;
+ public static final int field4 = 42;
+
+ public final int field5 = 42;          // violation, public immutable fields are allowed
+ public final java.lang.String notes;   // violation, public immutable fields are not allowed
+
+ public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;(); // violation, HashSet is mutable
+ public final ImmutableSet&lt;String&gt; mySet2;                // violation, mutable
+ public final ImmutableMap&lt;String, Object&gt; objects1;      // violation, mutable
```

Comment:
This is very weird, but in html them are very misaligned.
Please align them in html, as users use html most of the time.

Please recheck all other misalignments and better to align in html rather than in code

Reply:
you mean making them as:

```
<pre>
    <code>
        public final Set&lt;String&gt; mySet1 = new HashSet&lt;&gt;();     // violation, HashSet is mutable
        public final ImmutableSet&lt;String&gt; mySet2;                    // violation, mutable
        public final ImmutableMap&lt;String, Object&gt; objects1;      // violation, mutable
    </code>
</pre>
```
am i  right?

Reply:
No , keep same html structure as before, just remove spaces to make visually aligned in html.

In code they will be misaligned, it is ok.

Reply:
great, just making `mvn verify` and will push them right now.